Source/debug.cpp

```diff
@@ -249,10 +257,21 @@ std::string DebugCmdVisitTowner(const std::string_view parameter)
 std::string DebugCmdResetLevel(const std::string_view parameter)
 {
 	auto &myPlayer = Players[MyPlayerId];
-	auto level = atoi(parameter.data());
+
+	std::stringstream paramsStream(parameter.data());
+	std::string singleParameter;
+	if (!std::getline(paramsStream, singleParameter, ' '))
+		return "What level do you want to visit?";
+	auto level = atoi(singleParameter.c_str());
 	if (level < 0 || level > (gbIsHellfire ? 24 : 16))
 		return fmt::format("Level {} is not known. Do you want to write an extension mod?", level);
 	myPlayer._pLvlVisited[level] = false;
+
+	if (std::getline(paramsStream, singleParameter, ' ')) {
+		auto seed = atoi(singleParameter.c_str());

```

Comment:
Can this use `stoul` from `<string>`? `atoi` returns a signed int, this uses undefined behaviour for seeds larger than int_max.

```suggestion
		uint32_t seed = static_cast<uint32_t>(std::stoul(singleParameter));
```
(The cast isn't necessary on platforms where `sizeof(uint32_t) == sizeof(unsigned long)` but just to be safe)

Reply:
Didn't thought much about it, cause the old command line parameter did the same. 😉 
I opened #2746 to fix it.
Thank you 🙂